<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bat_information</class>
 <widget class="QWidget" name="bat_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/images/32_E3.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="change_equ_but">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>98</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="change_batinf_but">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>98</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="change_moni_but">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>98</y>
     <width>98</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="change_billinf_but">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>98</width>
     <height>81</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="bms_status">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>133</y>
     <width>98</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>待机</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bms_soc">
   <property name="geometry">
    <rect>
     <x>651</x>
     <y>133</y>
     <width>98</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rated_vol">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>173</y>
     <width>98</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="charge_timelife">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>213</y>
     <width>98</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="allow_charging_vol">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>294</y>
     <width>98</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="allow_charging_current">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>254</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>60</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="allow_charging_C">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>333</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>90</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_allow_charging_vol">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>374</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>90</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_vol_position">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>294</y>
     <width>98</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_measuring_C">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>333</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>90</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_vol">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>253</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>250</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bat_type">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>173</y>
     <width>98</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>锂</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bat_capacity">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>213</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_measuring_C_position">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>374</y>
     <width>98</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
